recitation_1/bash.md

```diff
@@ -136,22 +147,21 @@ $cut -d "delimiter" -f (field number) file.txt
 ```
 Lets's say we have a file called cut_demo.txt with some names in it;
 ```
+#This is the direct snapshot of a terminal, we will use the `$` symbol to differentiate it from codes in a text editor
 $ cat cut_demo.txt
-Dr. Borowski
-Leslie Chang
-Nguyen Tran
-$ cut -d " " -f2 cut_demo.txt     # delimiter: " " means a space character => we will break 'Dr. Borowski' into field 1:  'Dr.' and field2: 'Borowski'. 
-Borowski
-Chang
-Tran
+$ Dr. Borowski
+$ Leslie Chang
+$ Nguyen Tran
+$ cut -d " " -f2 cut_demo.txt     # delimiter: " " means a space character => we will use this to break 'Dr. Borowski' into field 1:  'Dr.' and field2: 'Borowski'. 
+$ Borowski
+$ Chang
+$ Tran
 ```
-### Basic pattern matching:
-TBA
 
 Exercise:
 ------
 ### Agenda:
-- First, we will go through Dr.B's "grade.sh" script in class. We will apply what we just learnt above to see what and why the code is doing what it does. You can obtain a copy of the code on Coursework.
+- First, we will go through Dr.B's "grade.sh" script in class. We will apply what we just learnt above to see what and why the code is doing what it does. A copy of the code should be available on Coursework.

```

Comment:
```suggestion
- First, we will go through Dr.B's "grade.sh" script in class. We will apply what we just learnt above to see what and why the code is doing what it does. A copy of the code should be available on Courseworks.

```

Reply:
@NguyenTran88 you should also ask Dr B if you can put the code into the recitation 1 folder. Would be best practice to have it all in one location. Any code we refer to in recitation should ideally be in the repo.